setup.cfg

```diff
@@ -5,6 +5,7 @@ author_email = mlodev@microsoft.com
 summary = Accera - An optimizing cross-compiler for compute-intensive code
 long_description = file: README.md
 long_description_content_type = text/markdown
+license = MIT License
```

Comment:
thanks for the fix! Can you also update these other setup.cfg files in the repo?

Accera/accera/python/compilers/setup.cfg
Accera/accera/python/gpu/setup.cfg
Accera/accera/python/llvm/setup.cfg

Cheers,
lisa